app/src/main/res/layout/activity_splash.xml

```diff
@@ -13,9 +13,13 @@
         android:src="@drawable/google"
         android:layout_width="200dp"
         android:layout_height="200dp" />
-    <ImageView
-        android:layout_width="80dp"
-        android:layout_height="65dp"
-        android:src="@drawable/books" />
+<!--    <ImageView-->
+<!--        android:layout_width="80dp"-->
+<!--        android:layout_height="65dp"-->
+<!--        android:src="@drawable/books" />-->
```

Comment:
remove comment if not needed, otherwise write reason why it is left

Reply:
Comment removed.